tests/integration/test_dbgap.py

```diff
@@ -148,15 +148,20 @@ def annotate_variable_using_babel_nemoserve(var_name, desc, permissible_values,
         first_result = result[0]
 
         denotation = dict(token)
+        denotation['id'] = f"{first_result['curie']}"
         denotation['text'] = text
         if method == 'sapbert':
             denotation['score'] = first_result['score']
-            denotation['name'] = first_result['name']
+            denotation['label'] = first_result['name']
             denotation['obj'] = f"{first_result['curie']} ({first_result['name']}, score: {first_result['score']})"
+            denotation['score'] = first_result['score']
```

Comment:
Line 157 can be removed since it is a duplicate of line 154

Reply:
Nice catch! Fixed in 85dd112.